src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenTests.cs

```diff
@@ -3231,7 +3231,7 @@ public static void Main()
     }
 }
 ";
-            var compilation = CompileAndVerify(source, expectedOutput: "222333");
+            var compilation = CompileAndVerify(source, options: TestOptions.ReleaseExe.WithModuleName("ＭＯＤＵＬＥ"), expectedOutput: "222333");
```

Comment:
Could we use a unicode escape sequence here vs. embedding unicode literals?  It confuses Visual Studio less.